app/models/mixins/supports_feature_mixin.rb

```diff
@@ -243,35 +246,52 @@ def feature_known?(feature)
       SupportsFeatureMixin::QUERYABLE_FEATURES.key?(feature.to_sym)
     end
 
+    def is_supported?(feature)
+      supported[feature] || superclass.try(:is_supported?, feature)
```

Comment:
After having to rearrange this code here to support minimal inheritance that did already exist, I am finding myself in complete agreement with @Fryguy about wanting to do this some form of real inheritance.

That said, I wasn't able to find exactly what he was referencing in the comments earlier:

https://github.com/ManageIQ/manageiq/pull/20417#issuecomment-669350124

So not sure how I would implement that.  Though, I am sure whatever it is better than the hack I have above.

Reply:
While I agree that the entire module could be reworked, I think this PR accomplishes the original goal - to get rid of the warnings while still passing its specs.